packages/forms/test/form_group_spec.ts

```diff
@@ -1825,6 +1825,225 @@ describe('FormGroup', () => {
     });
   });
 
+  describe('emit `statusChanges` and `valueChanges` with/without async/sync validators', () => {
+    describe('in stand alone', () => {
+      it('should run the async validator on stand alone controls and set status to `INVALID`',
+         fakeAsync(() => {
+           const logger: string[] = [];
+           const c =
+               new FormControl('', null, simpleAsyncValidator({timeout: 0, shouldFail: true}));
+           c.valueChanges.subscribe((value) => logger.push(`value: ${value}`));
+           c.statusChanges.subscribe((status) => logger.push(`status: ${status}`));
+
+           tick(1);
+
+           c.setValue('new!', {emitEvent: true});
+
+           tick(1);
+
+           expect(logger).toEqual(['status: INVALID', 'value: new!', 'status: INVALID']);
+         }));
+
+      it('should run the async validator on stand alone controls and set status to `VALID`',
+         fakeAsync(() => {
+           const logger: string[] = [];
+           const c = new FormControl('', null, asyncValidator('new!'));
+           c.valueChanges.subscribe((value) => logger.push(`value: ${value}`));
+           c.statusChanges.subscribe((status) => logger.push(`status: ${status}`));
+
+           tick(1);
+
+           c.setValue('new!', {emitEvent: true});
+
+           tick(1);
+
+           expect(logger).toEqual(
+               ['status: INVALID', 'value: new!', 'status: PENDING', 'status: VALID']);
+         }));
+
+      it('should run the sync validator on stand alone controls and set status to `INVALID`',
+         fakeAsync(() => {
+           const logger: string[] = [];
+           const c = new FormControl('new!', Validators.required);
+           c.valueChanges.subscribe((value) => logger.push(`value: ${value}`));
+           c.statusChanges.subscribe((status) => logger.push(`status: ${status}`));
+
+           tick(1);
+
+           c.setValue('', {emitEvent: true});
+
+           tick(1);
+           expect(logger).toEqual(['value: ', 'status: INVALID']);
+         }));
+
+      it('should run the sync validator on stand alone controls and set status to `VALID`',
+         fakeAsync(() => {
+           const logger: string[] = [];
+           const c = new FormControl('', Validators.required);
+           c.valueChanges.subscribe((value) => logger.push(`value: ${value}`));
+           c.statusChanges.subscribe((status) => logger.push(`status: ${status}`));
+
+           tick(1);
+
+           c.setValue('new!', {emitEvent: true});
+
+           tick(1);
+           expect(logger).toEqual(['value: new!', 'status: VALID']);
+         }));
+    });
+
+    describe('in combination', () => {
+      it('should run the async validator and set status to `VALID`', fakeAsync(() => {
```

Comment:
Let's also add a bit more info into the test description (for this test and other tests as well):
```suggestion
      it('should run the async validator on FormGroup and FormControl and set status to `VALID`', fakeAsync(() => {
```